index.html

```diff
@@ -2379,26 +2379,14 @@ <h4>Semantic search: SPARQL</h4>
 
                 <section id="directory-api-spec">
                     <h4>API Specification (Thing Model)</h1>
-
-                    <p class="issue" data-number="86">
-                        To do: add a
-                        <a href="https://w3c.github.io/wot-thing-description/#thing-model">Thing Model</a>
-                        (`directory.tm.json`)
-                        that can derive to
-                        <a href="https://github.com/w3c/wot-discovery/blob/main/implementations/example-directory-oauth2.td.json">this TD</a>
-                        as an example.
-                        The <a>TM</a> should describe all interaction affordances and indicate which ones
-                        are required (i.e. mandatory by this spec). Moreover, it should use placeholders
-                        for deployment-specific attributes.
-                        Informative affordances (JSONPath and XPath) should be defined as such.
```

Comment:
The issue isn't fully resolved. In particular, we still have the informative JSONPath and XPath in there.

I suggest we simply remove them since the TM is normative. The text already covers everything that is necessary for implementations. We could also move them to another TM but that's really unnecessary.

Reply:
We could also just add a comment to the TM, in the form of a "description", that the corresponding parts of the TM are not normative.